tools/diagnostics/full_bundle.py

```diff
@@ -63,9 +60,7 @@ def is_service_scheduler_task(package_name: str, service_name: str, task: dict)
     dcos_service_name = next(
         iter([l.get("value") for l in labels if l.get("key") == "DCOS_SERVICE_NAME"]), ""
     )
-    return dcos_package_name == package_name and dcos_service_name == to_dcos_service_name(
-        service_name
-    )
+    return dcos_package_name == package_name and dcos_service_name == service_name
```

Comment:
Should this not also use the same logic as for `is_service_named`?

Reply:
This value seems to always have the leading slash, even though in the Mesos state the service name might not.